be1-go/hub/standard_hub/hub_state/queries.go

```diff
@@ -20,17 +22,21 @@ type Queries struct {
 // NewQueries creates a new queries struct
 func NewQueries() Queries {
 	return Queries{
-		state:                  make(map[int]*bool),
+		state:                  make(map[int]bool),
```

Comment:
I just realized something, also completely outside the scope of this PR.
Does this map grow forever and ever as the application keeps running ? 
When do we clear out old IDs to free up some resources ? 😅 
We might need to add some mechanism for that.

Reply:
The map does indeed grow forever ... I just created #1707 to open the discussion about this so that it's better documented. Let me know what you think !